lib/verdict/railtie.rb

```diff
@@ -5,10 +5,15 @@ class Verdict::Railtie < Rails::Railtie
     Verdict.default_logger = Rails.logger
 
     Verdict.directory ||= Rails.root.join('app', 'experiments')
-    app.config.eager_load_paths -= Dir[Verdict.directory.to_s]
 
-    # Re-freeze eager load paths to ensure they blow up if modified at runtime, as Rails does
-    app.config.eager_load_paths.freeze
+    if Rails.respond_to?(:autoloaders) && Rails.autoloaders.zeitwerk_enabled?
```

Comment:
```suggestion
    if Rails.autoloaders.zeitwerk_enabled?
```

I don't think we need to be defensive about this, do we? If rails is only partially loaded, we want this to fail.

Reply:
Or is this a compatibility issue? I would do a version check to make it easier to discern why we need different code paths.

Reply:
Yeah `autoloaders` was added at the same time than zeitwerk (rails 6).
I'll do the version check 👍

Reply:
Done !